packages/widget-chat/src/reducers/conversation.js

```diff
@@ -4,7 +4,8 @@ function conversation(state = {
   activities: [],
   id: null,
   participants: [],
-  isFetching: false
+  isFetching: false,
+  isLoaded: false
```

Comment:
It probably isn't necessary, I just liked having an explicit "state" (sorry @ianwremmel) to check against. My thinking being, the conversation object might change requiring the check to change as well.